DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/AutocadConverterToHost.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Autodesk.AutoCAD.DatabaseServices;
+using Speckle.Autofac.DependencyInjection;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.Autocad;
+
+public class AutocadConverterToHost : ISpeckleConverterToHost
+{
+  private readonly IFactory<string, ISpeckleObjectToHostConversion> _toHost;
+  private readonly IHostToSpeckleUnitConverter<UnitsValue> _unitConverter;
```

Comment:
Is this used?

Reply:
No, will remove from this class and also the `AutocadConverterToSpeckle` class as well. Should probably also remove it from Rhino, as it isn't used there as well